Formula/pjproject.rb

```diff
@@ -25,7 +25,7 @@ class Pjproject < Formula
   depends_on "openssl@1.1"
 
   def install
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}", "--disable-g722-codec"
```

Comment:
Does this not just work on Darwin? Is it an upstream bug? Did it work previously, or is it a new codec?

Reply:
Turns out to have been a parallelization issue.